front-end/src/components/MainPage/MainPage.js

```diff
@@ -3,16 +3,27 @@ import PropTypes from 'prop-types'
 import { Container, Content, Button, Text } from 'native-base'
 
 const MainPage = ({ navigation }) => {
-  const handlePress = () => {
-    navigation.navigate('LoanList')
+  const handlePress = name => {
+    navigation.navigate(name)
   }
 
   return (
     <Container>
       <Content>
-        <Button onPress={handlePress}>
+        <Button
+          onPress={() => {
+            handlePress('LoanList')
+          }}
+        >
           <Text>Go to the homepage of Loan Section</Text>
         </Button>
+        <Button
+          onPress={() => {
+            handlePress('ProductSelection')
```

Comment:
@puteulanus instead of navigating from the MainPage let's wire this into the new loan's page when clicking on the selected product